manager-bundle/src/Resources/skeleton/config/security.yml

```diff
@@ -76,7 +76,7 @@ security:
                 prepare_on_access_denied: true
 
             remember_me:
-                secret: '%secret%'
+                secret: '%env(APP_SECRET)%'
```

Comment:
souldn't we use `kernel.secret` here?

Reply:
Probably yes.

Reply:
Changed in e383840bbcaee2179e51e8532c608afebcc02991.